templates/solutions/telco/index.html

```diff
@@ -16,32 +16,30 @@
 
 {% block content %}
 
-  <section class="p-strip--shallow">
-    <div class="p-section">
-      <div class="row--50-50 p-section">
-        <div class="col">
-          <h1>Transform your infrastructure with secure and cloud-native telecom solutions</h1>
-        </div>
-        <div class="col">
-          <p>
-            Canonical is the trusted and agile partner you need to boost network performance, optimise costs and achieve digital transformation with open source.
-          </p>
-          <p>Get enterprise-grade security and support to innovate with confidence.</p>
-          <hr class="p-rule--muted" />
-          <a class="p-button--positive"
-             href="https://ubuntu.com/telco#get-in-touch">Contact us</a>
-        </div>
+  <section class="p-section--hero">
+    <div class="row--50-50 p-section">
+      <div class="col">
+        <h1>Transform your infrastructure with secure and cloud-native telecom solutions</h1>
       </div>
-      <div class="u-fixed-width u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/d3517134-Asset%205@2x.png",
-                alt="",
-                width="500",
-                height="376",
-                hi_def=True,
-                loading="auto") | safe
-        }}
+      <div class="col">
+        <p>
+          Canonical is the trusted and agile partner you need to boost network performance, optimise costs and achieve digital transformation with open source.
+        </p>
+        <p>Get enterprise-grade security and support to innovate with confidence.</p>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive"
+            href="https://ubuntu.com/telco#get-in-touch">Contact us</a>
       </div>
     </div>
+    <div class="u-fixed-width u-align--center">
```

Comment:
I believe this should be within p-image-section

Reply:
I don't think this is needed. From the [docs](https://vanillaframework.io/docs/patterns/images), the image container is meant to "Enhance images by distinguishing them from their background or controlling their container's aspect ratio." This iso isn't supposed to be on a different background color and the aspect ratio is already correct as is